nutils/function.py

```diff
@@ -736,8 +753,9 @@ def evalf(cls, *args: Any) -> numpy.ndarray:
 
         raise NotImplementedError  # pragma: nocover
 
-    @classmethod
-    def partial_derivative(cls, iarg: int, *args: Any) -> IntoArray:
+    @types.hashable_function('NotImplemented')
+    @staticmethod
```

Comment:
staticmethod is implied by hashable_function

Reply:
fixed